contracts/ceros/provider/BaseLpTokenProvider.sol

```diff
@@ -0,0 +1,265 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.10;
+
+import "@openzeppelin/contracts-upgradeable/security/ReentrancyGuardUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import "../interfaces/IVault.sol";
+import "../interfaces/IDex.sol";
+import "../interfaces/IDao.sol";
+import "../interfaces/IHelioProviderV2.sol";
+import "../interfaces/IBNBStakingPool.sol";
+import "../interfaces/ICertToken.sol";
```

Comment:
[Better to import specific names instead of the whole modules?](https://ethereum.stackexchange.com/questions/117100/why-do-many-solidity-projects-prefer-importing-specific-names-over-whole-modules)

for example: 
```
import {ICertToken} from "../interfaces/ICertToken.sol";
```